### PR TITLE
SE-1092 move fastcgi_params

### PR DESCRIPTION
**Pull request by bmxmiko**
https://berlingskemedia.atlassian.net/browse/SE-1092
I've moved fastcgi_params to bm.env.symfony repo

### CODE REVIEW

File: fastcgi_params [hunk 1]
```diff
@@ -1,28 +0,0 @@
-# default setup
-fastcgi_param  QUERY_STRING       	$query_string;
```
Comment (bmxlugo) [inline fastcgi_params]: @bmxmiko why did you remove all of these?

Comment (bmxmiko) [inline fastcgi_params]: @bmxlugo Because I moved all of these params to bm.env.symfony repo, I think there is better place for this. https://github.com/BerlingskeMedia/bm.env.symfony/pull/123/files#diff-646c7f61a632806a77a754aaef0be734
